### DISCUSSION TITLE
Will templ be able to form a desktop app?

### DISCUSSION THREAD

**Discussion by Purple-CSGO** [a-h/templ#398] [Ideas]
The way of writing golang directly with html is quite insane and clear, and I'm wondering if this can be used to form a desktop app?

1. Combine Webview2 and some desktop tools to build up apps
2. Combine PWA app and the GUI is actually a PWA app with golang as the backend

> kind of brainstorming, but really looking forward to some cool stuff :)

**Comment by a-h** [comment]
Interesting idea. I've seen https://github.com/wailsapp/wails but that uses Vite. Maybe it's possible to switch out the frontend for SSR.

The webview_go bindings look pretty straightforward: https://github.com/webview/webview_go/tree/master/examples/basic

**Comment by meaLuda** [comment]
I would be interested in this can we meet and brainstorm ?
